src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -1,5 +1,22 @@
 package org.stellar.walletsdk
 
+import org.stellar.sdk.KeyPair
+import org.stellar.sdk.Network
+import org.stellar.sdk.Server
+import shadow.com.google.gson.annotations.SerializedName
+
 class Wallet(private val horizonUrl: String, private val horizonPassphrase: String) {
-    // TODO: Wallet code goes here
-}
\ No newline at end of file
+  private val server = Server(this.horizonUrl)
+  private val network = Network(this.horizonPassphrase)
+
+  data class AccountKeypair(
+    @SerializedName("publicKey") val publicKey: String,
+    @SerializedName("secretKey") val secretKey: String
+  )
+
+  // Create account keys, generate new keypair
+  fun create(): AccountKeypair {
+    val keypair: KeyPair = KeyPair.random()
+    return AccountKeypair(keypair.accountId, String(keypair.secretSeed))
```

Comment:
The `AccountKeypair.publicKey` can be derived from `AccountKeypair.secretKey`, so we don't need to pass both to the constructor.

Reply:
Wouldn't we be adding extra step developers would need to do after `Wallet.create()` method is called? It feels more convenient to return both keys.

Reply:
Oh we should definitely make sure the `AccountKeypair` object provides devs with both, but what I'm saying is that we don't need to pass both keys to the constructor. We can pass `secretKey` to the object, and should be able to fetch `publicKey` & `secretKey` values from it.

Reply:
The keypair objects have methods `fromSecretSeed()` and and `fromPublicKey()`. Maybe we should have a similar pattern.

Reply:
Let's keep it as is for now. We can change it later if we feel strongly about it.